cluster-migration/src/main/java/com/wipro/ats/bdre/clustermigration/SourceStageLoad.java

```diff
@@ -0,0 +1,97 @@
+/*
+ *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.wipro.ats.bdre.clustermigration;
+
+import com.wipro.ats.bdre.BaseStructure;
+import com.wipro.ats.bdre.IMConfig;
+import com.wipro.ats.bdre.im.IMConstant;
+import com.wipro.ats.bdre.md.api.GetProperties;
+import org.apache.commons.cli.CommandLine;
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+/**
+ * Created by cloudera on 4/12/16.
+ */
+public class SourceStageLoad extends BaseStructure{
+
+    private static final Logger LOGGER = Logger.getLogger(MigrationPreprocessor.class);
+    private static final String[][] PARAMS_STRUCTURE = {
+            {"stgDb", "src-stage-db", "Source stage database"},
+            {"stgTbl", "src-stg-table", " Source stage table"},
+            {"stgPts", "stg-part-cols", "Source Stg partition columns"},
+            {"srcCols", "src-reg-cols", "Source stage regular columns"},
+            {"stgBp", "stg-bp-cols", " Business partition columns"},
+            {"execId", "instance-exec-id", "Instance execution id"},
+            {"srcDb", "source-db", " Source database"},
+            {"srcTbl", "source-table", "Source table"},
+            {"filCon", "filter-condition", "Filter condition"},
+            {"ppid", "parent-process-id", "Parent Process Id"}
+    };
+
+    protected static Connection getHiveJDBCConnection(String dbName, String hiveConnection) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00112)